packages/react-components/src/components/MessageThread.tsx

```diff
@@ -752,6 +744,14 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
     [new Date().toDateString()]
   );
 
+  const localeStrings = useLocale().strings.messageThread;
+  const strings = useMemo(() => ({ ...localeStrings, ...props.strings }), [localeStrings, props.strings]);
```

Comment:
Not for this PR.

This pattern is likely required in every component that takes `strings`. This is a good candidate for a generic `useX` function.